wp-content/themes/bpr2018/resources/assets/sass/_footer.scss

```diff
@@ -67,21 +95,20 @@
     padding-left: 25px;
   }
 
-  hr {
-    position: absolute;
-    margin-top: 1px;
-    margin-left: -38;
+  #sub_heading {
+    font-size: 20px;
+    margin-top: 20px;
```

Comment:
replace `20px` with `$spacing-sm` since we already have a sass variable for this